Ideas-2025.md

```diff
@@ -95,4 +102,218 @@ Specifically, we would be working toward accomplishing the following:
 **Coding Mentors**<br>
 [Walter Bender](https://github.com/walterbender/)<br>
 **Assisting Mentors**<br>
-[Devin Ulibarri](https://github.com/pikurasa/)
\ No newline at end of file
+[Devin Ulibarri](https://github.com/pikurasa/)
+
+------------
+
+## Add an AI-assistant to the Write Activity
+
+**Prerequisites**<br>
+ - Experience with Python
+ - Experience with Sugar activities
+ - Experience with LLMs/Chatbots
+
+**Description**<br>
+
+Sugar pioneered peer editing in its Write activiy. However, the Write
+Activity has never had any serious support for grammar correction
+(just spell check) and none of the more recent developments around
+AI-assisted writing. The goal of this project is to add AI-assistance
+to the writing process: both in the form of providing feedback as to
+what has been written and making suggestions as to what might be
+written.
+
+The challenge will be both in terms of workflow integration and UX.
+
+**Project Length**<br>
+
+**350** hours
+
+**Difficulty**<br>
+
+**High**
+
+**Coding Mentors**<br>
+[Walter Bender](https://github.com/walterbender/)
+[Ibiam Chihurumnaya](https://github.com/chimosky/)
+
+**Assisting Mentors**<br>
+
+------------
+
+## Refactor the Infoslicer Activity to generate plain-language summaries
+
+**Prerequisites**<br>
+ - Experience with Python
+ - Experience with Sugar activities
+ - Experience with LLMs/Chatbots
+
+**Description**<br>
+
+The Infoslicer Activity is designed to help teachers extract content from the Wikipedia in order to c    reate lesson plans. This is currently a manual, extractive process. It is well suited to generative A    I. The goal would be to have a teacher type in a theme for a lesson and have the AI create a simple l    esson plan, which the teacher can then edit.
+
+The biggest challenge to summarization using generative AI is hallucinations. A work-around for this     is to include a validation step that surfaces evidence (or lack of evidence) for each assertion in th    e lesson plan. This will introduce some workflow and UX challenges.
+
+**Project Length**<br>
+
+**350** hours
+
+**Difficulty**<br>
+
+**High**
+
+**Coding Mentors**<br>
+[Walter Bender](https://github.com/walterbender/)
+[Ibiam Chihurumnaya](https://github.com/chimosky/)
+
+**Assisting Mentors**<br>
+
+------------
+
+ ## Refactor the chatbot in the Speak Activity to use AI
```

Comment:
"gen-AI"

The current chatbot does have a kind of "AI", just using conditionals